spec/checkPullRequestReviewSpec.js

```diff
@@ -77,6 +75,26 @@ describe('Pull Request Review Module', () => {
       };
     });
 
+    describe('LGTM Label is there', ()=>{
```

Comment:
Update test description to something like when reviewer requests changes and LGTM label was already added to the pull request.

Reply:
Sure.